src/schema-parser/avro-schema-parser.ts

```diff
@@ -0,0 +1,354 @@
+import avsc from 'avsc';
```

Comment:
Didnt schema parsers get released separately and removed from the parser repo?

Reply:
For some reason, those are back here but they are removed from `master` branch 🤷

Reply:
PR created removing them again https://github.com/asyncapi/parser-js/pull/802